src/api/graphql/typeBuilder.ts

```diff
@@ -36,13 +39,23 @@ const types: {
 } = {}
 
 export function buildFields(table: ITableInfo, options: IBuildTypeOptions = { relations: true } ) {
```

Comment:
please make the pagination true by default.

Reply:
I've changed the pagination to true by default but most don't use pagination:
![Screenshot from 2020-01-24 15-20-52](https://user-images.githubusercontent.com/6785689/73080085-293fa780-3ebd-11ea-84c4-ddf860343827.png)